.gitignore

```diff
@@ -3,5 +3,4 @@ vendor
 .php.cache
 .phpunit.result.cache
 .phpunit.cache
-composer.lock
```

Comment:
Inderdaad 🤯

Reply:
Weet nu niet of dit positief of negatief moet zien 🤔 

Maar in iedergeval anders klapte die security check telkens, het is natuurlijk ook de vraag of je composer.lock wil bij houden in je versie beheer daar zijn de mening nog best over verdeelt

Reply:
In je eigen project moet je `composer.lock` altijd mee committen. Degene die de package gebruikt, gebruikt alleen zijn eigen .lock file. Maar voor het runnen van de tests in deze repo is het wel goed om alle versies te locken.